README.md

```diff
@@ -87,6 +89,8 @@ $GOPATH/src/github.com/go-clang/gen/scripts/create-clang-version.sh 3.4
 
 This will create a new repository `v3.4` in the `$GOPATH/src/github.com/go-clang` directory and initialize it using the `bootstrap` repository. The command also generates, installs, configures and tests bindings for the given Clang version. The changes must then be manually verified, added, committed and pushed to the already set up remote "origin".
 
+After the repository is online godoc.org needs to know about the new repository. This can be achieved by visiting [https://godoc.org/github.com/go-clang/v&lt;MAJOR&gt;.&lt;MINOR&gt;/clang](https://godoc.org/github.com/go-clang/v<MAJOR>.<MINOR>/clang).
```

Comment:
couldn't this be done automatically by the generation script?
```
$> curl https://godoc.org/github.com/go-clang/vM.N/clang
```
and be done with it?